diff --git a/include/pci_general/pci_general.h b/include/pci_general/pci_general.h
index 0a955fc..d0850fd 100644
--- a/include/pci_general/pci_general.h
+++ b/include/pci_general/pci_general.h
@@ -24,4 +24,5 @@ namespace explorer {
-
+typedef Eigen::Matrix<double, 5, 1> StateVec;
+
 class PCIGeneral : public PCIManager {
  public:
   PCIGeneral(const ros::NodeHandle &nh, const ros::NodeHandle &nh_private);
